tests/specs/pa/bs-pa.ts

```diff
@@ -0,0 +1,121 @@
+import * as assert from 'assert';
+
+import { BS } from '../po/bs-po';
+import { bsData } from '../test-data/bs-data';
+
+const bs = new BS();
+
+class BSActions {
+
+    async CreateBucket() {
+
+       await bs.AddSpace_Button.click();
+       await bs.SpaceName_Field.setValue(bsData.SpaceName);
+       await bs.Create_Button.waitForClickable({
+         timeout: 2000,
+       });
+       await bs.Create_Button.click();
+
+    }
+
+    async UploadFile() {
+
+        await bs.Bucket_Link.click();
+        const remoteFilePath = await browser.uploadFile('../tests/specs/test-files/Logo.svg');
+        await bs.File_Input.setValue(remoteFilePath);
+        await bs.Reload_Button.waitForClickable({
+          timeout: 2000,
+        });
+        await bs.Reload_Button.click();
+ 
+    }
+
+    async DeleteFile() {
+
+        await bs.Bucket_Link.click();
+        await bs.Bucket.waitForClickable({
+          timeout: 2000,
+        });
+        await bs.Bucket.click();
+        await bs.ConfirmDeleting.waitForClickable({
+            timeout: 2000,
+          });
+          await bs.ConfirmDeleting.click();
+ 
+    }
+
+    async CheckFileExist() {
+        await $('tr>td[class="styles_tableCell__C6dnh"]').waitForExist({
+            timeout: 2000,
+          });
+        let tableCell = await $('tr>td[class="styles_tableCell__C6dnh"]');
+        let tableCellValue = await tableCell.getText();
+        let fileCreated;
+        if (bsData.FileName == tableCellValue){
+           fileCreated = true;
+        }
+        else{
+           fileCreated = false;
+        }
+        assert.equal(fileCreated,true);
+    }
+
+    async CheckFileDeleting() {
+        let tableCell = await $('tr>td[class="styles_tableCell__C6dnh"]');
```

Comment:
```suggestion
        let tableCell = await $('tr>td[class="styles_tableCell__C6dnh"]');
```
bad styles. let discuss it on the call